docs-content/release-notes.html.md.erb

```diff
@@ -9,8 +9,11 @@ owner: Partners
 Features included in this release:
 
 * Client information moved from Evolven to PCF.
+  - Credentials for Ops Manager and for PKS is stored now in PCF (instead of Evolven in the previous version).
+  - Url for Ops and CF api are set on the tile setting page.
 * Collect incrementally the configuration and the applications.
-* New name as given to the release and as result changed the version as well. 
+  - Evolven agent will collect the applications content from Apps Manager only if they were modified since the last execution.
+* New name as given to the release and as result changed the version as well.
 
 ##<a id="ver"></a> v6.2.7

```

Comment:
Seems like v6.2.7 has duplicate entries, not sure when this was introduced by might be worth cleaning up.